src/components/Modals/SaveCourseModal.vue

```diff
@@ -17,19 +17,24 @@
       <!-- create a rectangular border-->
       <div class="saveCourseModal-header-text">
         <span> Collections </span>
-        <button class="saveCourseModal-header-text-addButton" @click="addNewCollection">
+        <button class="saveCourseModal-header-text-addButton" @click="addCollection">
           <img src="@/assets/images/plus.svg" alt="add new collection" />
         </button>
       </div>
     </div>
 
     <div class="saveCourseModal-body">
       <div class="saveCourseModal-body-content">
-        <p>{{ collection }}</p>
-        <!--Must find all possible collections
-            Checkbox Style
-            Need a collections variable in firestore
-        -->
+        <div v-for="collection in collections" :key="collection">
+          <input
+            v-if:="!isDefaultCollection"
+            type="checkbox"
+            id="collection"
+            :value="collection"
+            v-model="checkedCollections"
+          />
+          <label for="collection">{{ collection }}</label>
```

Comment:
Heads up, I noticed when testing that this label will always only toggle the very first collection when clicked. So e.g. the label for the fourth collection, when clicked, would toggle the checkmark for first collection.

I suspect that this is due to the non-uniqueness for `id` and `for` — could perhaps combine this with the indices or use `"collections"` literal value?

Additionally, would it be possible to have a little margin between the checkbox and the label? I appreciate that this is a draft PR though 👍 .